### DISCUSSION TITLE
Prisma2 Deployment

### DISCUSSION THREAD

**Discussion by adetorodev** [prisma/prisma#13546] [Q&A]
I just created my app using Prisma, ApolloServer, and TypeScript. 
I am ready for deployment to Heroku. I don't really know how to deploy the schema.prisma file. 
Will that deploy with my node app to heroku?
Most of what I found online were prisma1

**Comment by nurul3101** [comment]
Hey @adetoroe787 👋 
Here's the guide for Deploying to Heroku: [Guide](https://www.prisma.io/docs/guides/deployment/deployment-guides/deploying-to-heroku)
You can follow this guide to deploy your application.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
